spec/support/api/schemas/payment.json

```diff
@@ -0,0 +1,31 @@
+{
+  "type": "object",
+  "required": [
+    "agency_id",
+    "amount_cents",
```

Comment:
are the `amount_currency` and `discrepancy_currency` also technically required?

Reply:
nope.  removed